### ISSUE TITLE
Feature: Robin Office Pass Configuration

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#6114]
Met with Gilda to review her re entry schedule and discuss the Robin Office Pass

- Gilda has a spreadsheet of when each employee at Barton Oaks is scheduled to come into the office
- For those who will be hoteling when they come into the office we discussed setting up a Office Pass
- Gilda and Stephen will review the schedule and determine which Office Passes need to be set up
- Stephen will add employees to the appropriate Office Pass 

- [ ] Set up the Office Passes based on the re entry schedule

**Comment by moorerst** [comment]
All, in the meeting we discussed using the Robin Office Pass to manage the staggering schedule for staff. I looked at the spreadsheet Gilda provided and there are 12 different combinations of hoteling work schedules. Just a reminder of the Admin task related to managing Office Pass we would need to create a Office Pass for each possible work schedule and add the staff to that Office Pass.
If someone attempted to schedule a hotel desk outside of their Office Pass schedule they would receive a message that they are not scheduled to be in the office.
In my opinion setting up and managing 12 different Office Passes could be challenging. Keeping everyone in the default Office Pass would have less overhead and gives staff the ability to reserve a hotel desk on any day.
Let me know your thoughts on moving forward with the Office Pass option.
